extensions/shared/icons.scss

```diff
@@ -0,0 +1,22 @@
+@import './styles/gutenberg-base-styles.scss';
+@import '../../_inc/client/scss/variables/colors';
```

Comment:
You won't be able to import those colors here, as the file isn't available on WordPress.com. Can we pull those colors from somewhere else?

Reply:
I didn't realize this! Would you be okay copying colors that are used into that file? Otherwise I have no idea how to do this and know very little about the rest of Jetpack to judge if I can move files around

Reply:
I think it'd be okay, but I would encourage you to add comments in both places so folks who want to update the colors in one place know that they have to update it in both places.

It'd be nice if we could pull those colors from a core package or another @automattic package, but I don't know if we have something like that already.

Reply:
Removed import and rolled back to using colors from the colors studio. The only downside is that we don't have any color for Google Search, only the orange of Google+. For now, I have used the orange as the closest thing we have available without updates to any dependencies.

Reply:
Doesn't `@import './styles/gutenberg-base-styles.scss';` just import `@wordpress/base-styles`? Can't we make that a dependency and import them that way?

The `_inc/client/scss/variables/colors` is a problem though I guess.